fmralign/alignment_methods.py

```diff
@@ -249,4 +249,4 @@ def fit(self, X, Y):
     def transform(self, X):
         """Transform X using optimal coupling computed during fit.
         """
-        return self.R.dot(X)
+        return X.dot(self.R)
```

Comment:
Do you have a specific test to check that these operations are done correctly ?

Reply:
Yes after finding a bug in the sense of the multiplication, I added a test of prediction shape to test_alignment_method to be sure that the product was done in the right way.